src/main/java/com/smattme/MysqlExportService.java

```diff
@@ -338,14 +347,32 @@ private String exportToSql() throws SQLException {
         TablesResponse allTablesAndViews = MysqlBaseService.getAllTablesAndViews(database, stmt);
 
         List<String> tables = allTablesAndViews.getTables();
-        //for every table, get the table creation and data
-        // insert statement
-        for (String s: tables) {
-            try {
-                sql.append(getTableInsertStatement(s.trim()));
-                sql.append(getDataInsertStatement(s.trim()));
-            } catch (SQLException e) {
-                logger.error("Exception occurred while processing table: " + s, e);
+
+        // Retrieve specific tables from properties, if they exist
+        String specificTables = properties.containsKey(SPECIFIC_TABLES_FOR_EXPORT)

```

Comment:
We can simplify this statement by using `String specificTables = properties.getProperty(SPECIFIC_TABLES_FOR_EXPORT, "");` instead of the existing ternary operator